tools/picard/picard_AddCommentsToBam.xml

```diff
@@ -1,4 +1,4 @@
-<tool name="AddCommentsToBam" id="picard_AddCommentsToBam" version="1.126.0">
+<tool name="AddCommentsToBam" id="picard_AddCommentsToBam" version="1.135">
```

Comment:
picard will update eventually, and as galaxy is using the underlying tool version, I don't suppose you'd be willing to encapsulate this in a macro?

E.g. https://github.com/galaxyproject/tools-iuc/blob/master/tools/bedtools/macros.xml#L8 is a macro which is then used like this: https://github.com/galaxyproject/tools-iuc/blob/master/tools/bedtools/bed12ToBed6.xml#L1